.github/workflows/tpu-tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       shell: bash
 
     # https://docs.github.com/en/actions/deployment/deploying-to-your-cloud-provider/deploying-to-google-kubernetes-engine
-    - uses: google-github-actions/get-gke-credentials@fb08709ba27618c31c09e014e1d8364b02e5042e
+    - uses: google-github-actions/get-gke-credentials@v0
```

Comment:
This change introduced a warning, please fix it before merging: https://github.com/Lightning-AI/lightning/actions/runs/3311982177/jobs/5468098497#step:8:21

Reply:
This PR will be ready for merging once this job runs successfully: https://github.com/Lightning-AI/lightning/actions/runs/3316015384/jobs/5477312282#logs